server/homebrew.api.js

```diff
@@ -116,7 +116,10 @@ const api = {
 		return text;
 	},
 	getGoodBrewTitle : (text)=>{
-		const tokens = Markdown.marked.lexer(text);
+		let tokens = Markdown.marked.lexer(text);
+		while (tokens[0].type == 'pageBlock'){
```

Comment:
Why not an `if` here? Is it possible to nest pages that we need to loop through more than once?

Reply:
It _shouldn't_ be possible for the first token to be anything other than `pageBlock`. I seem to recall using `if` and having it not work, but I don't recall any specific details. I'll switch to `if` and report if problems arise.